Sources/ArgumentParser/Parsing/InputOrigin.swift

```diff
@@ -103,6 +103,12 @@ struct InputOrigin: Equatable, ExpressibleByArrayLiteral {
   }
 }
 
+extension InputOrigin {
+  var isDefaultValue: Bool {
+    return _elements.count == 1 && _elements.first == .defaultValue
```

Comment:
Maybe there is a better way to do this? I also tried a `contains` and `Set` comparison but this seemed the simplest, easiest to read, and (assuming) most performant choice.